src/uu/mv/src/error.rs

```diff
@@ -4,19 +4,23 @@
 // file that was distributed with this source code.
 use std::error::Error;
 use std::fmt::{Display, Formatter, Result};
+use std::path::PathBuf;
 
+use uucore::display::Quotable;
 use uucore::error::UError;
 
 #[derive(Debug)]
 pub enum MvError {
     NoSuchFile(String),
+    CannotStatNotADirectory(String),
     SameFile(String, String),
     SelfSubdirectory(String),
     SelfTargetSubdirectory(String, String),
     DirectoryToNonDirectory(String),
     NonDirectoryToDirectory(String, String),
     NotADirectory(String),
     TargetNotADirectory(String),
+    FailedToAccessNotADirectory(PathBuf),
```

Comment:
Would it be better to store a `PathBuf` instead of a `String` for all error variants?

Reply:
Yes, I don't know why `String` is used. Though I think it is outside the scope of this PR.

Reply:
Thinking the same, wanted to point it out though. Will revert to string for now.